schedule-2019-03-internal.md

```diff
@@ -0,0 +1,95 @@
+# Instructor Training (Internal, March 2019)
+
+Copyright 2019, RStudio Inc.
+All material made available under the [Creative Commons Attribution-NonCommercial 4.0](https://creativecommons.org/licenses/by-nc/4.0/) license.
+
+## Instructors
+
+- Garrett Grolemund
+- Greg Wilson
+
+## Day 1
+
+1.  Getting Started (0:15/09:15)
+
+1.  Two Ways to Teach (0:45/10:00)
+    -   <./keynote/10-two-ways-to-teach.key>
+    -   Comparison with movie/play (add comedy improv)
+    -   Tell participants there's lots of research (much of which people don't know/act on)
+    -   Explain what active learning is (creates a real-time feedback loop in class)
+    -   Introduce cognitive architecture diagram (revisit it several times)
+        -   Teaching is "load into short-term memory and keep it there long enough to be transferred"
+    -   Bloom's Taxonomy
+        -   Google isn't enough if you don't know what to search for or how to recognize a useful answer when you find one
+        -   GG added "awareness" to the bottom of the Bloom hierarchy since it's the most realistic goal for a workshop like this
+    -   Exercise: classify three learning objectives by level
+
+1.  Novice/Competent Practitioner/Expert (0:45/10:45)
+    -   <./keynote/20-novice-competent-expert.key>
+    -   Mental models
+        -   Emphasize linkages: we rarely forget completely, but we lose the ability to retrieve
+    -   Mental models don't have to be right to be useful (ball-and-spring models in chemistry)
+    -   How the graph model of human cognition expains the difference between competent and experts
+        -   Experts have more connections
+    -   Concept maps as visual representations of mental models
+    -   Using concept maps:
+        -   To design lesson
+        -   To convey information to learners
+        -   For insight into their learning
+    -   Exercise: draw and compare concept maps for ggplot2
+
+1.  Coffee (0:15/11:00)
+
+1.  Teaching as Performance I (0:30/11:30)
+    -   <./keynote/30-teaching-as-performance-1.key>
+    -   Critique the Nederbragt videos
+        -   Bad: <https://youtu.be/bXxBeNkKmJE>
+        -   Good: <https://youtu.be/SkPmwe_WjeY>
+    -   Exercise: teach in pairs (no recording)
+    -   Build rubric for use in second teaching session
+
+1.  Multimedia Learning (0:30/12:00)
+    -   <./keynote/80-multimedia-learning.key>
+    -   Exercise: create a visual to explain what the course is about
```

Comment:
I wonder if a more useful exercise/complementary exercise would be to critique a standard text heavy slide deck.

Reply:
I think that would be a soft target at this point :-) I've changed the exercise to "create a visual to explain a dplyr pipeline" - please switch it to something else if you have a better idea (because I'm sure better ideas are possible).

Reply:
I wish it were a soft target, but it still what people do. There's at least a couple recent offenders in our class.